homeassistant/components/switch/rpi_rf.py

```diff
@@ -0,0 +1,94 @@
+"""
+Allows to configure a switch using a 433MHz module via GPIO on a Raspberry Pi.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.rpi_rf/
+"""
+
+import logging
+
+from homeassistant.components.switch import SwitchDevice
+from homeassistant.const import CONF_VALUE_TEMPLATE
+
+REQUIREMENTS = ['RPi.GPIO==0.6.2', 'rpi-rf==0.9.5']
+
+_LOGGER = logging.getLogger(__name__)
+
+
+# pylint: disable=unused-argument
+def setup_platform(hass, config, add_devices_callback, discovery_info=None):
+    """Find and return switches controlled by a generic RF device via GPIO."""
+    import rpi_rf
+
+    gpio = config.get('gpio', 17)
```

Comment:
Is 17 a port that is mostly right or is that arbitrary? If the latter we should make it a mandatory config option. This will make it more clear to the user what is going on.